src/FilamentBreezyServiceProvider.php

```diff
@@ -61,8 +65,22 @@ public function packageBooted(): void
             });
         }
 
-        ResetPassword::createUrlUsing(function ($user, string $token) {
-            return route('filament.password.reset', ['token' => $token]);
+        ResetPassword::createUrlUsing(function ($notifiable, string $token) {
+            return route('filament.password.reset', [
+                'token' => $token,
+                'email' => $notifiable->getEmailForPasswordReset(),
+            ]);
+        });
+
+        VerifyEmail::createUrlUsing(function ($notifiable) {
```

Comment:
same here.

I will test more once I got some time.
but for now, this addition is following the prev. one.
so it's safe to be applied.